runtime/slot-dom.js

```diff
@@ -0,0 +1,106 @@
+/**
+ * @license
+ * Copyright (c) 2017 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+"use strict";
+
+const assert = require('assert');
+const Slot = require('./slot.js');
+
+class DomSlot extends Slot {
+  constructor(slotid) {
+    super(slotid);
+    this._dom = null;
+  }
+  initialize(context, exposedView) {
+    this._dom = context;
+    // TODO(sjmiles): what is this for?
+    this.exposedView = exposedView;
```

Comment:
exposedView is the view that's declared in the particle spec:
https://github.com/PolymerLabs/arcs/blob/master/particles/ListView/ListView.ptcl#L14